pkg/await/extensions_ingress.go

```diff
@@ -50,6 +51,7 @@ type ingressInitAwaiter struct {
 	endpointsReady   bool
 	endpointsSettled bool
 	endpointExists   map[string]bool
```

Comment:
A better name for this is now probably something like `serviceReady`, as `Service`s with type `ExternalName` don't have endpoints.

Reply:
Arg, dammit. I'm testing out the GH extension and it apparently doesn't let you batch up review comments?